README.md

```diff
@@ -69,7 +69,7 @@ Options:
 
 `markdown-inject` expands a given glob for markdown files. Then it discovers the below `CODEBLOCK` HTML comments within each markdown file, performs the appropriate action (in this case, reading another local file), and writes content back into the markdown file:
```

Comment:
I think this paragraph should mention CODEBLOCK HTML or MDX comments, as the syntax is distinct